trans/views/translation.py

```diff
@@ -35,8 +35,14 @@ def get(self, request, *args, **kwargs):
         home_flat_page = FlatPage.objects.filter(slug=home_flat_page_slug).first()
         home_content = home_flat_page.content if home_flat_page else ''
         translating_users = User.objects.exclude(groups__name__exact=['staff', 'editor'])
+        current_translating_user = None
         if not user.is_editor():
-            translating_users = translating_users.select_related('language', 'country').exclude(language__code__exact='en').exclude(country__user=user).order_by('country__name')
+            translating_users = translating_users.select_related('language', 'country').order_by('country__name')
+            if (user.language.code != 'en'):
+                current_translating_user = translating_users.get(country__user=user, language__user=user)
```

Comment:
Hmm, shouldn't `current_translating_user` be always equal to `user`? Basically it's the currently logged-in user?